src/core/tools/common/thrift_message_parser.h

```diff
@@ -60,6 +60,10 @@ namespace dsn
     // response format:
     //     <total_len(int32)> <thrift_string> <thrift_message_begin> <body_data(bytes)> <thrift_message_end>
 
+# define THRIFT_HDR_SIG (*(uint32_t*)"THFT")
```

Comment:
Why only define macro for THRIFT? or we can just use *(uint32_t*)"THFT" like others.